spec/javascripts/content_tools_links_spec.js

```diff
@@ -0,0 +1,37 @@
+describe("PopupView.generateContentToolsLinks", function () {
+  var renderingApplication = "smartanswers";
+  var PROD_ENV = {}
+
+  it("returns the URL to the GovSpeak file", function () {
+    PROD_ENV.url = "https://www.gov.uk/smart-answer/y/question-1";
+
+    var deferred = $.Deferred();
```

Comment:
@tijmenb I stubbed it so that I wouldn't actually have to visit the page in the tests. Is there a better way to do this?